### PR TITLE
AVIARY-2023 custom fields tables column

### PR DESCRIPTION
**Pull request by furqanwasi**
AVIARY-2023 custom fields tables column

### CODE REVIEW

File: app/models/organization.rb [hunk 1]
```diff
@@ -8,6 +8,8 @@ class Organization < ApplicationRecord
   has_many :billing_histories, dependent: :destroy
   has_many :collections, dependent: :destroy
   has_many :organization_users, dependent: :destroy
+  has_many :organization_emails, dependent: :destroy
+  has_one :theme, dependent: :destroy
```
Comment (smntb) [inline app/models/organization.rb]: there is no existance of theme, please delete

File: app/models/organization.rb [hunk 2]
```diff
@@ -27,6 +29,7 @@ class Organization < ApplicationRecord
   validates_attachment_content_type :favicon, content_type: %r[\Aimage\/(png|jpeg|jpg|x-icon|vnd.microsoft.icon)\z]
   validates_attachment_content_type :logo_image, :banner_image, content_type: %r[\Aimage\/(png|gif|jpeg|jpg)\z]
   accepts_nested_attributes_for :user
+  accepts_nested_attributes_for :organization_emails
```
Comment (smntb) [inline app/models/organization.rb]: why this list is added?

**Review by smntb — CHANGES_REQUESTED** [review]

File: app/models/organization.rb (hunk 1 above)
Comment (furqanwasi) [reply app/models/organization.rb]: already fixed it in coming PRs forgot to change in this one

File: app/models/organization.rb (hunk 2 above)
Comment (furqanwasi) [reply app/models/organization.rb]: already fixed it in coming PRs forgot to change in this one

**Review by smntb — APPROVED** [review]

**Pull request merged**